app/templates/my-sessions/view.hbs

```diff
@@ -0,0 +1,40 @@
+<div class="ui container">
+  <div class="ui row">
+    <button class="ui blue button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Edit Session Proposal'}}</button>
+    {{#if device.isMobile}}
+      <div class="ui hidden fitted divider"></div>
+    {{/if}}
+    <button class="ui blue button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Edit Speaker Details'}}</button>
+    {{#if device.isMobile}}
+      <div class="ui hidden fitted divider"></div>
+    {{/if}}
+    <button class="ui red button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Withdraw Proposal'}}</button>
+    {{#if device.isMobile}}
+      <div class="ui hidden fitted divider"></div>
+    {{/if}}
+  </div>
+  <div class="ui row">
+    <h2 class="ui header left aligned">
+      {{t 'Session Details'}}
+    </h2>
+    <div class="ui segment">
+      <h3 class="ui header">
+        {{model.state}}
```

Comment:
@ritikamotwani Use label here for the states. Make sure the colors are dynamically displayed depending upon the state of the session. 

Resource : https://semantic-ui.com/elements/label.html